.github/workflows/hil_sample_zephyr.yml

```
@@ -219,6 +220,19 @@ jobs:
             twister-out/*.xml
             twister-out/*.json
 
+      - name: Prepary report summary
+        if: always() && steps.mask-logs.outcome == 'success'
+        run: |
+          mkdir summary
+          cp twister-out/twister_suite_report.xml summary/twister-${{ inputs.hil_board }}.xml
+
+      - name: Upload report summary
+        uses: actions/upload-artifact@v4
+        if: always() && steps.mask-logs.outcome == 'success'
+        with:
+          name: ci-summary-twister-${{ inputs.hil_board }}
```

```suggestion
          name: ci-summary-samples-zephyr-${{ inputs.hil_board }}
```

Use an artifact title that more closely follows the workflow names